riot-headers.h

```diff
@@ -184,6 +184,14 @@
 #include "ws281x.h"
 #endif
 
+/* wolfSSL */
+#ifdef MODULE_WOLFSSL
```

Comment:
```suggestion
#if defined(MODULE_WOLFSSL) && !defined(IS_C2RUST)
```

While RIOT is "sloppy" about API and ABI, WolfSSL being a regular library should have all its interfaces bindgen-safe. If we start exposing external library's symbols (which I think we'll need to, because a regular -sys crate won't know the build flags and include paths the way we know them), making them bindgen-only should become best practice where possible.